tests/v2/integration/rtbs/rtbs_test.go

```diff
@@ -80,7 +81,8 @@ func (p *RTBTestSuite) TestPRTBRoleTemplateInheritance() {
 	client, err := p.client.WithSession(subSession)
 	require.NoError(p.T(), err)
 
-	createdNamespace, err := extnamespaces.CreateNamespace(client, namegen.AppendRandomString("testns-"), "{}", map[string]string{}, map[string]string{}, p.project)
+	projectID := strings.Split(p.project.ID, ":")[1]
```

Comment:
is `p.project.ID` going to be clusterID:projectID? I ask because why dont we send the project directly to the extension and split this to get project and clusterID in the extension itself? then we can avoid doing the same everytime we use CreateNamespace? Let me know what you think.

Then we can keep the function as is and make changes in the extension itself.

Reply:
yes, that's correct. `p.project.ID` is `clusterID:projectID`. I can add it to the extension

Reply:
I realized now why I had not included it in the extension :) `projectID` format is different in norman versus steve/public API.
- In norman -  `"id": "c-m-zd6c22wb:p-ckbht",`
- In steve - `"id": "c-m-zd6c22wb/p-ckbht",`
- In public API -` id is not available `

To avoid confusion, I modified the extension now to take `projectName` as input instead of projectID.